INFO     schema_assertion:schema_assertion.py:16 Loaded schema: create_team_schema.json
INFO     schema_assertion:schema_assertion.py:26 JSON schema validation for 'create_team_schema.json' passed.
INFO     team_service:team_service.py:20 Starting the process to create a team.
INFO     auth_service:authentification.py:10 Initializing Auth class.
INFO     auth_service:authentification.py:17 Loading credential configuration from 'credentials.json'.
INFO     auth_service:authentification.py:13 Auth class initialized successfully.
INFO     auth_service:authentification.py:37 Building headers for user type: 'invalid_user'
INFO     auth_service:authentification.py:28 Fetching user details for user type: 'invalid_user'
INFO     auth_manager:auth_manager.py:18 Building headers for user: invalid
INFO     auth_manager:auth_manager.py:9 Encoding credentials for user: invalid
INFO     auth_manager:auth_manager.py:28 Headers build successfully for user: invalid
INFO     auth_service:authentification.py:47 Headers successfully built for user type: 'invalid_user'
INFO     api_manager:api_request.py:23 POST Request URL: https://testing.espocloud.com/api/v1/Team | {'Espo-Authorization': 'aW52YWxpZDppbnZhbGlk', 'X-Api-Key': 'f542db99f09e823eeb14cd558f104fef', 'Content-Type': 'application/json'}
INFO     api_manager:api_request.py:25 Response Status Code: 401
INFO     team_service:team_service.py:24 Finished processing the create team request.
INFO     status_code_assertion:status_code_assertion.py:47 Validating status code 401 for response with URL: https://testing.espocloud.com/api/v1/Team
INFO     status_code_assertion:status_code_assertion.py:52 Status code 401 validated successfully.